public/locales/en/categories.json

```diff
@@ -2,9 +2,9 @@
     "a5": "A 5",
     "activationfunctions": "Activation Functions",
     "ahocorasick": "Aho-Corasick",
+    "ai": "AI",
```

Comment:
Was this file modified manually or automatically?

Reply:
That is automatic